src/sentry/api/serializers/models/dashboard.py

```diff
@@ -89,14 +89,27 @@ def get_attrs(self, item_list, user):
 
             result[dashboard]["widget_preview"].append(widget_preview)
 
+        user_serializer = UserSerializer()
+        serialized_users = {
+            user["id"]: user
+            for user in serialize(
+                [dashboard.created_by for dashboard in item_list if dashboard.created_by],
```

Comment:
Should the users be eager loaded with a `select_related`?

Reply:
We do select related on the users here: https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_dashboards.py#L49
Will look at the prefetch_related_objects

Reply:
Just got a chance to look at this again, both DiscoverSavedQuery and Dashboards select_related("created_by") in the initial queryset, so the prefetch shouldn't be necessary. I'm not sure if the preferred pattern is to do the select_related in the initial query or the prefetch_related_objects in the serializer. I'll leave it as is for now, I can update if there's a string preference for one/other

Reply:
I generally recommend doing `prefetch_related_objects ` in the serializers regardless of what outside classes are doing. It won't cause extra work if the related objects have already been fetched. But it means that if someone else uses this serializer in another endpoint they don't have to remember to call `select_related` on it.